docs/gitbook/examples/to-do-list.md

```diff
@@ -1,425 +1,257 @@
 
-#### Actions
+### Set Up and Configuration
 
-Actions serve two purposes:
-* They are dispatched to the chain during operation, and passed to the Smart Contracts to change the Application State.
-* They are passed to the chain on initialization to inform it of which operations are valid.
+We named our app `app-todo-list` and updated our public and private covenant names correspondingly. See:
 
-To ensure that we always have a consistent Action format, we describe our Action types and use a set of functions to generate our Actions with a consistent format. This file is called `actions.js` by convention, and is located in `/src/actions.js` in this example.
+- `package.json`
+- `interbit.config.js`
+- `interbit.prod.config.js`
 
-Once the Action types are defined and passed to the chain on initialization, you can dispatch them to the chain, which then passes them through the Smart Contracts with the current state.
+The public, private, and control chain configurations are exactly as they are in the template app&mdash;all we've done is rename things here. The template app is configured so that users can dispatch actions to their private chain, so our To Do List app does the same. User's have their own private to do lists. In the future, we may extend this example so that users can have shared lists.
 
-<div class="filename">src/actions.js</div>
+### Modifying the Private Covenant
 
-```js
-let actions = {
-  ADD_ITEM: 'ADD_ITEM',
-  addItem: (id, label, description) => {
-    return {
-      type: actions.ADD_ITEM,
-      payload: {
-        id,
-        label,
-        description
-      }
-    }
-  },
-  // ...
-}
-```
+Our To Do List app allows users to add items, edit items, and mark items as complete in their to do list. These actions are described in the app's private covenant in `src/interbit/private`. We've removed the actions included in the Template app and added the `ADD_TODO`, `EDIT_TODO`, and `TOGGLE_TODO` actions.
 
-<div class="tips info">
-  <p><span></span>The <code>ADD_ITEM</code> Action Type and Creator</p>
-  <p>In the snippet above, an Action type and creator function for that item are shown. The creator function accepts parameters we need to process this Action, so that when we make an Action anywhere in our application or tests, we know we are always supplying the Smart Contract with the required payload.</p>
-</div>
+Actions are dispatched to the chain during operation, and are passed to the [smart contract](../key-concepts#smart-contracts) to change the [application state](../key-concepts#application-state). Interbit follows the Redux design pattern, so actions in the Interbit covenenants are essentially the same as actions in Redux. All the same, we'll go through each of the files in `src/interbit/privagte`.
```

Comment:
The links to the key concepts sub-topics should include the slash indicating that it's a directory. For example: `(../key-concepts/#smart-contracts)`.

Fix spelling of "covenenants" to "covenants".

Fix spelling of "privagte" to "private".